electron/legendary/games.ts

```diff
@@ -279,13 +279,13 @@ Categories=Game;
     const enabledInDesktop = GlobalConfig.get().config.enableDesktopShortcutsOnDesktop
     const enabledInStartMenu = GlobalConfig.get().config.enableDesktopShortcutsOnStartMenu
 
-    if (enabledInDesktop || enabledInDesktop === undefined) {
```

Comment:
we should ask the user to define it instead in the future. Else this feature won't be used.

Reply:
I just added a parameter to pass when creating the shortcuts from the submenu. so it will be automatically if the feature is enabled or if manually, it will pass a parameter for that.